cvat/apps/annotation/builtin/pascal_voc/dumper.py

```diff
@@ -0,0 +1,33 @@
+from pascal_voc_writer import Writer
+import os
+from zipfile import ZipFile
+from tempfile import TemporaryDirectory
+
+def dumps_as_pascal_voc(file_object, image_dir, annotations):
+    with TemporaryDirectory() as out_dir:
+        with ZipFile(file_object, 'w') as output_zip:
+            for frame_annotation in annotations:
+                image_name = frame_annotation.name
+                width = frame_annotation.width
+                height = frame_annotation.height
+
+                image_path = os.path.join(image_dir, image_name)
+                writer = Writer(image_path, width, height)
+                writer.template_parameters['path'] = image_dir
+
+                for shape in frame_annotation.shapes:
+                    if shape.type != "rectangle":
```

Comment:
I see that Pascal VOC supports polygons, boxes, pixel masks, etc. Are we going to support them?

Reply:
I'm going to significantly refactor pascal format support when I add pascal parser. The same about using pascal_voc_writer package.